source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -279,65 +283,98 @@ def image(
 class image_features(ManagerTermBase):
     """Extracted image features from a pre-trained frozen encoder.
 
-    This method calls the :meth:`image` function to retrieve images, and then performs
-    inference on those images.
+    This term uses models from the model zoo in PyTorch and extracts features from the images.
+
+    It calls the :func:`image` function to get the images and then processes them using the model zoo.
+
+    A user can provide their own model zoo configuration to use different models for feature extraction.
+    The model zoo configuration should be a dictionary that maps different model names to a dictionary
+    that defines the model, preprocess and inference functions. The dictionary should have the following
+    entries:
+
+    - "model": A callable that returns the model when invoked without arguments.
+    - "reset": A callable that resets the model. This is useful when the model has a state that needs to be reset.
+    - "inference": A callable that, when given the model and the images, returns the extracted features.
+
+    If the model zoo configuration is not provided, the default model zoo configurations are used. The default
+    model zoo configurations include the models from Theia and ResNet. These models are loaded from
```

Comment:
Maybe link Theia here too? https://theia.theaiinstitute.com/

Reply:
Added citations to the papers.